src/Components/Artwork/GridItem.tsx

```diff
@@ -57,14 +58,8 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
       return null
     }
 
-    let breakpoint
     if (this.state.isMounted) {
-      Object.keys(breakpoints).forEach(key => {
-        if (breakpoints[key]) {
-          breakpoint = key
-        }
-      })
-
+      const breakpoint = Object.keys(pickBy(breakpoints))[0]
```

Comment:
What’s the resulting type of breakpoint?

Reply:
Got it. I now see that this is in a method where `breakpoints` isn’t typed at all, so that means the typing isn’t going to be any better than `string` anyways.

I don’t even know if TS supports it nowadays, but theoretically it could warn when a `switch` clause doesn’t handle a possible value in the union/enum.

So this is all just out of interest.

Reply:
btw there's a `default` for the switch below (which is a sane default for the legacy contexts).